airbyte-integrations/connectors/source-github-singer/Dockerfile.test

```diff
@@ -0,0 +1,24 @@
+FROM airbyte/base-python-test:dev
+
+RUN apt-get update && apt-get install -y \
+    vim \
```

Comment:
does vim need to be here? i think this ships with `vi` but I could be wrong

Reply:
nope. me debugging. will remove.